### ISSUE TITLE
mkdocs warnings

### ISSUE THREAD

**Issue by fblanqui** [Deducteam/Logipedia#39]
```
07:46 ~/src/Logipedia (master) mkdocs build
INFO    -  Cleaning site directory 
INFO    -  Building documentation to directory: /home/blanqui/src/Logipedia/site 
INFO    -  The following pages exist in the docs directory, but are not included in the "nav" configuration:
  - authors.md
  - erratum.md
  - structure.md
  - export/apropos.md
  - export/cli.md
  - export/installation.md
  - import/apropos.md
  - import/contributing.md
  - publication/generation.md
  - publication/installation.md 
WARNING -  A relative path to 'import.md' is included in the 'nav' configuration, which is not found in the documentation files 
WARNING -  A relative path to 'translation.md' is included in the 'nav' configuration, which is not found in the documentation files 
WARNING -  A relative path to 'export.md' is included in the 'nav' configuration, which is not found in the documentation files 
WARNING -  Documentation file 'index.md' contains a link to 'manifesto.md' which is not found in the documentation files. 
WARNING -  Documentation file 'index.md' contains a link to 'export/implem.md' which is not found in the documentation files. 
WARNING -  Documentation file 'index.md' contains a link to 'publication/apropos.md' which is not found in the documentation files. 
WARNING -  Documentation file 'index.md' contains a link to 'publication/implem.md' which is not found in the documentation files. 
```

**Comment by gabrielhdt** [comment]
Hi Amélie, you can :+1:

**Comment by francoisthire** [comment]
I don't remember those files. I think @gabrielhdt must have added them.

**Comment by gabrielhdt** [comment]
Fixed in #40
